src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -67,9 +63,8 @@ private GameDataManager() {}
   public static GameData loadGame(final File file) throws IOException {
     checkNotNull(file);
 
-    try (final FileInputStream fis = new FileInputStream(file);
-        final InputStream is = new BufferedInputStream(fis)) {
-      return loadGame(is, getPath(file));
+    try (final InputStream is = new BufferedInputStream(new FileInputStream(file))) {
```

Comment:
Depending on how you split hairs between checked and unchecked exceptions, this may not be safe.  If the `BufferedInputStream` constructor throws an exception (and it can only throw an unchecked exception), the `FileInputStream` will not be closed.

One could argue that all bets are off if an unchecked exception is thrown and that, in that case, it doesn't matter if the underlying stream is closed or not.  However, IMO, applying Automatic Resource Management consistently (i.e. one variable per `AutoCloseable` resource) seems preferable than having to know when it's safe to skip it or not.